.github/workflows/gnovm.yml

```diff
@@ -5,9 +5,6 @@ on:
     branches:
       - master
   pull_request:
-    paths:
```

Comment:
I'd say find the dependency, and add it as a path requirement

Reply:
I would say just about the entire repository; maybe an exception for `tm2/*`.